core/src/types.ts

```diff
@@ -166,6 +191,31 @@ export interface IL2VrfFulfillListenerWorker {
   randomWords: string[]
 }
 
+export interface IL2RequestResponseListenerWorker {
+  callbackAddress: string
+  blockNum: string
+  requestId: string
+  jobId: string
+  accId: string
+  callbackGasLimit: number
+  sender: string
+  numSubmission: number
+  /* eslint-disable  @typescript-eslint/no-explicit-any */
+  req: any
+}
+
+export interface IL2RequestResponseFulfillListenerWorker {
+  callbackAddress: string
+  blockNum: number
+  requestId: string
+  jobId: string
+  callbackGasLimit: number
+  sender: string
+  l2RequestId: string
+  /* eslint-disable  @typescript-eslint/no-explicit-any */
+  response: any
```

Comment:
`response` can either be `BigInt`, `string` or `boolean` which can be expressed as 

```suggestion
  response: BigInt | string | boolean
```

However, I am a bit doubtful your solution currently works for `BigInt` because BullMQ is not able to automatically serialize `BigInt`. Having `any` only hides the problems. Once we merge this PR, we should tests with `uint256` request.

Reply:
I will test with every datatypes after deployment. One more option we ca consider is using `number` instead of `bigint`